examples/camera-controller/webrtc_manager/WebRTCProviderClient.cpp

```diff
@@ -139,11 +139,11 @@ CHIP_ERROR WebRTCProviderClient::SendCommandForType(Messaging::ExchangeManager &
     }
 }
 
-void WebRTCProviderClient::OnDeviceConnectedFn(void * context, Messaging::ExchangeManager & exchangeMgr,
-                                               const SessionHandle & sessionHandle)
+void WebRTCProviderClient::OnDeviceConnected(void * context, Messaging::ExchangeManager & exchangeMgr,
+                                             const SessionHandle & sessionHandle)
 {
     WebRTCProviderClient * self = reinterpret_cast<WebRTCProviderClient *>(context);
-    VerifyOrReturn(self != nullptr, ChipLogError(NotSpecified, "OnDeviceConnectedFn: context is null"));
+    VerifyOrReturn(self != nullptr, ChipLogError(NotSpecified, "OnDeviceConnected: context is null"));
 
     ChipLogProgress(NotSpecified, "CASE session established, sending WebRTCTransportProvider command...");
     CHIP_ERROR sendErr = self->SendCommandForType(exchangeMgr, sessionHandle, self->mCommandType);
```

Comment:
After sending the ProvideOffer, the controller should move to a state(say, ProvideOfferSent) and have some timeout for the Answer on that session to arrive.

Reply:
Acked, https://github.com/project-chip/connectedhomeip/issues/38260